.github/workflows/test.yml

```diff
@@ -0,0 +1,50 @@
+name: Test
+
+on:
+  workflow_dispatch:
+    inputs:
+      os-variant:
+        default: 'ubuntu-24.04'
+        type: string
+        description: 'Operating system'
+      python-version:
+        type: string
+        description: 'Python interpreter version'
+      tox-env:
+        default: 'test'
+        type: string
+        description: 'Tox environment to run'
+      checkout_ref:
+        default: ''
+        type: string
+        description: 'Git ref to checkout'
+  workflow_call:
+    inputs:
+      os-variant:
+        default: 'ubuntu-24.04'
+        type: string
+      python-version:
+        type: string
+      tox-env:
+        default: 'test'
+        type: string
+      checkout_ref:
+        default: ''
+        type: string
```

Comment:
Maybe we should also include a scicat backend version input in the workflow_dispatch and workflow_call? so that when we trigger the workflow we can trigger the workflow with specific version, e.g, v4.8.0 or latest or stable

Reply:
Yes!

Do we need to make more changes to make it callable from the backend project?

Reply:
I think we can start with this workflow and see how it works together. I will add a workflow for this on scicat backend next week as I have quite a few tickets on hand at the moment.

Reply:
I added it. You should be able to trigger the workflow now. Let me know how it goes!

Reply:
will do, thanks!